src/file/file.ts

```diff
@@ -132,18 +134,6 @@ export class File {
                 this.footnotesWrapper.View.createFootNote(parseFloat(key), options.footnotes[key].children);
             }
         }
-
-        if (options.features) {
-            if (options.features.trackRevisions) {
```

Comment:
Is this getting added back anywhere?

Reply:
Otherwise Demo 60 won't work?

https://github.com/dolanmiu/docx/blob/730e33b1644e541c18aab6413794220eb20edab9/demo/60-track-revisions.ts

Reply:
src/file/settings/settings.ts

Reply:
Rather than calling this.settings.addTrackRevisions(), it's passed straight as an option into the Settings object which handles it in the constructor.